content/docs/2.9/scalers/loki.md

```diff
@@ -0,0 +1,165 @@
++++
+title = "Loki"
+availability = "v2.9+"
+maintainer = "Community"
+description = "Scale applications based on Loki query result."
+go_file = "loki_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `loki` trigger that scales based on a Loki query result. Here is an example of providing values in metadata:
+
+```yaml
+triggers:
+- type: loki
+  metadata:
+    # Required fields:
+    serverAddress: http://<loki-host>:3100 # Note: loki server URL 
+    metricName: syslog_write_total # Note: name to identify the metric, generated value would be `loki-syslog_write_total`
+    query: sum(rate({filename="/var/log/syslog"}[1m])) by (level) # Note: query must return a vector/scalar single element response
+    threshold: '0.7'
+    # Optional fields:
+    activationThreshold: '2.50'
+    cortexOrgId: my-org # Optional. X-Scope-OrgID header for Cortex.
+    ignoreNullValues: false # Default is `true`, which means ignoring the empty value list from Loki. Set to `false` the scaler will return error when Loki target is lost
+    unsafeSsl: false #  Default is `false`, Used for skipping certificate check when having self signed certs for Loki endpoint
+```
+
+**Parameter list:**
+
+- `serverAddress` - URL of Loki server.
+- `metricName` - Name to identify the metric in the external.metrics.k8s.io API. If using more than one trigger it is required that all `metricName`(s) be unique.
```

Comment:
As in core PR, we don't support setting metricName in new scalers because it's an internal thing, old scalers have it but we are trying to remove it